release_tester/arangodb/installers/tar.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+""" run an installer for the debian based operating system """
+import sys
+import time
+import os
+import shutil
+import logging
+from pathlib import Path
+import psutil
+from arangodb.instance import ArangodInstance
+from arangodb.installers.base import InstallerBase
+from tools.asciiprint import ascii_print
+logging.basicConfig(level=logging.INFO, format='%(asctime)s %(message)s')
+import tools.loghelper as lh
+import semver
+
+
+class InstallerTAR(InstallerBase):
+    """ install .deb's on debian or ubuntu hosts """
+    def __init__(self, cfg):
+
+        self.tar = 'tar'
```

Comment:
add the platform-library at the start of the file: `import platform`
and here add the branch to choose the proper tar command:
```
    macver = platform.mac_ver()
    if macver[0]:
```
use `gtar` - aka GNU-tar on mac.